app/models/user.rb

```diff
@@ -8,4 +8,8 @@ def self.from_omniauth(auth)
           user.email = auth.info.email
         end
     end
+
+    def name
```

Comment:
samson wants us to replace this with "#{first_name} #{last_name}"

see his comment: https://github.com/saasbook/berkeley-reentry-student-program/pull/6#discussion_r853373007